src/sass/components/_path.scss

```diff
@@ -20,7 +20,31 @@ $color-background-path-complete-hover: $color-gray-6;
 .slds-path__nav {
   flex: auto;
 
-  .slds-is-active:not(.slds-is-won) {
+  .slds-is-complete {
+    background: $color-background-path-complete;
+    flex-grow: 0.5;
+
+    &:before,
+    &:after {
+      background: $color-background-path-complete;
```

Comment:
Would it work for `:before` & `:after` to inherit the background color? Then you could change it one place on hover, without changing it both places.